docs/integrations/delta.md

```diff
@@ -20,7 +20,53 @@ Because lakeFS is format-agnostic, you can save data in Delta format within a la
 
 {% include toc.html %}
 
-## Beta: Viewing Delta table changes in lakeFS 
+## Beta: Viewing Delta Lake table changes in lakeFS 
+
+### Enable the Delta Lake tables diff
+
+The Delta Lake diff functionality is enabled using a binary plugin. This binary plugin can be found as part of the latest lakeFS
+[release](https://github.com/treeverse/lakeFS/releases/latest), and can also be built locally using the `make delta-plugin` command
+(run from lakeFS's repository root). After getting it, place it under `~/.lakefs/plugins/diff`.  
+lakeFS will look for a binary named `delta_diff` under `~/.lakefs/plugins/diff` if not configured otherwise.  
+
+#### Customizing the Delta Lake diff plugin location
```

Comment:
I would put it last. Of all the things that I can do with the delta-diff functionality, changing the plugin location is the least interesting of all.